bundles/org.openhab.binding.homewizard/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -98,6 +131,68 @@
 
 	</thing-type>
 
+	<thing-type id="energy_socket">
```

Comment:
This thing type is already defined in line 78?

Reply:
Merge conflicts leftover. Fixed.